nethogs.cpp

```diff
@@ -128,10 +128,10 @@ int process_tcp (u_char * userdata, const dp_header * header, const u_char * m_p
 	switch (args->sa_family)
 	{
 		case (AF_INET):
-			packet = new Packet (args->ip_src, ntohs(tcp->source), args->ip_dst, ntohs(tcp->dest), header->len, header->ts);
+			packet = new Packet (args->ip_src, ntohs(tcp->th_sport), args->ip_dst, ntohs(tcp->th_dport), header->len, header->ts);
```

Comment:
Jep, those are synonymous in netinet/tcp.h (union with fields of the same type).

Presumably for compatibility with linux/tcp.h, but I have no problems with depending on netinet specifics.